Assets/SC_Cube.cs

```diff
@@ -0,0 +1,28 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class SC_Cube : MonoBehaviour
+{
+    private Transform transform;
+
+    public float rotationspeed;
+    // Start is called before the first frame update
+    void Start()
+    {
+        transform = GetComponent<Transform>();
+    }
+
+    //Today i had onigiri for lunch
+    void Update()
+    {
+        rotationspeed = //good work
+            Time.deltaTime;
+        transform
+            .Rotate(90*
+                    rotationspeed, 
+                0, 
+                0
+                );
+    }
```

Comment:
- All unnecessary comments should be removed
- You should not split the code into multiple lines when it can be written in one. It is difficult for me to understand what exactly is written.
- You should not calculate rotationSpeed in the Update method, but instead look for better ways to get a value for rotationSpeed, since it is not efficient to calculate it this way.